.github/contributing.md

```diff
@@ -1,3 +1,160 @@
+
+- `npm start` runs Astro's Local Development Server, listening by default on `http://localhost:4321/`.
+- `npm run build` builds the Application on Production mode. The output is by default within `dist` folder.
+  - This is used for the Netlify Deployments (Preview & Production)
+- `npm run preview` runs Astro's Local Server pointed at the `dist` folder.
+
+</details>
+
+<details>
+  <summary>Commands for Maintenance Tasks and Tests</summary>
+
+- `npm run ts-check` runs Astro's typescript check utility to look for any type issues.
+- `npm run format` formats and fixes the whole codebase
+
+</details>
+
+## Commit Guidelines
+
+This project follows the [Conventional Commits][] specification.
+
+Commits should be signed. You can read more about [Commit Signing][] here.
+
+### Commit Message Guidelines
+
+- Commit messages must include a "type" as described on Conventional Commits
+- Commit messages **must** start with a capital letter
+- Commit messages **must not** end with a period `.`
+
+## Pull Request Policy
+
+This policy governs how contributions should land within this repository. The lines below state the checks and policies to be followed before merging and in the act of merging.
+
+### Before merging
+
+We recommend a read on our [Collaborator Guide](collaborating.md#accepting-modifications) for in-depth details on how we accept contributions into this repository.
+The list below describes some merging and approval rules adopted in this repository.
+
+- Pull Requests must be open for at least 48 hours (Or 72 hours if the PR was authored on the weekend).
+  - Pull requests might be immediately merged if they contain critical bug fixes, short errata (e.g., typos from previous PRs), or any critical change considered a "showstopper" for the website's operation.
+    - This kind of PRs should only be done by existing collaborators with write-access and/or signed off by administrators/maintainers.
+    - This rule cannot be used for updates on the `collaborating.md`, `CONTRIBUTING.md` guide, GitHub Actions, or any security-impacting file or document that changes the governing policies of this repository.
+  - Pull requests might be "fast-tracked", meaning they can be merged before the usual 48 hours' notice if a "fast-track" label is added.
+    - The person that is fast-tracking the PR (adding the label) must also comment on the PR that they're requesting the PR to be fast-tracked
+    - Fast-tracking is only allowed for small bug fixes, small feature changes, localization changes, or other non-critical/highly-impacting changes not covered by the previous rule that allows PRs to be merged immediately.
+    - Fast-tracking cannot be used for updates on the `collaborating.md`, `CONTRIBUTING.md` guide, GitHub Actions, or any security-impacting file or document that changes the governing policies of this repository.
+- There must be no objections after forty-eight (48) hours (Or seventy-two (72) hours if the PR was authored on the weekend).
```

Comment:
This seems a little overkill to me. I'm curious where the idea came from. Just my two cents.

Reply:
It came from nodejs

Reply:
I agree it's a bit much. Happy to slash it down a bit